### PR TITLE
Translated for portuguese Collections

### PR DESCRIPTION
**Pull request by tavaresgerson**

### CODE REVIEW

File: pt/core-libraries/collections.rst [hunk 1]
```diff
@@ -1,12 +1,1222 @@
...
+
+    // O resultado ficará assim quando convertido em array
+    [
+        'a' => [1 => 'foo', 3 => 'baz'],
+        'b' => [2 => 'bar']
+    ];
+
+Por fim, você pode usar *closures* para criar caminhos de chaves/valores/grupos dinamicamente, 
+por exemplo, ao trabalhar com entidades e datas (convertidas em instâncias ``Cake/Time`` pelo ORM), 
+você pode agrupar os resultados por data
```
Comment (othercorey) [inline pt/core-libraries/collections.rst]:
```suggestion
você pode agrupar os resultados por data::
```

File: pt/core-libraries/collections.rst [hunk 2]
```diff
@@ -1,12 +1,1222 @@
...
+         ['2012', '200', '300', '400'],
+         ['2013', '100', '200', '300'],
+         ['2014', '50', '100', '200'],
+     ]
+
+.. versionadded:: 3.3.0
+    ``Collection::transpose()`` was added in 3.3.0.
+
+Retirando Elementos
+--------------------
```
Comment (othercorey) [inline pt/core-libraries/collections.rst]:
```suggestion
-------------------
```

**Comment by othercorey** [comment]
@tavaresgerson ok to merge?

**Comment by tavaresgerson** [comment]
> @tavaresgerson ok to merge?

Yes :-)

**Comment by othercorey** [comment]
Thanks!

**Pull request merged**